torch/_dynamo/utils.py

```diff
@@ -1689,24 +1689,24 @@ def defake(x):
 untouched_torch_utils_checkpoint = torch.utils.checkpoint.checkpoint
 
 
-def higher_order_op_converter():
+def is_utils_checkpoint(obj):
+    return (
+        obj is torch.utils.checkpoint.checkpoint
+        or obj is untouched_torch_utils_checkpoint
+    )
```

Comment:
Just to remind myself: we do this because we need to torch.dynamo.disable inside of torch.utils.checkpoint.checkpoint? Could we just put the torch.dynamo.disable call directly inside of torch.utils.checkpoint.checkpoint?

Reply:
Yes, ideally, I would like to put this disable directly on `utils.checkpoint`. Currently, it causes circular imports. I am trying to solve circular imports issue first. Once that effort lands, I will come back and clean this up.